src/components/Dropdown/Dropdown.stories.mdx

```diff
@@ -99,6 +102,9 @@ export default SimpleDropdown;
 Given the nature of a composite component, in addition to its own, the Dropdown utilizes CSS variables from the Button, Input and List components.
 They are listed in the table below.
 
+Use the CSS variable ```--dropdown-list-top``` to control the list position.<br />
```

Comment:
@viniciusaa can you please make an adjustment to this observation text. We should clarify that both variables are used to control the positioning, however, one of them will be ideal for a given listPosition property.